crates/holochain_metrics/src/lib.rs

```diff
@@ -65,7 +65,7 @@
 //!             .with_description("tx5 signal server request duration")
 //!             .with_unit(Unit::new("s"))
 //!             .init();
-//!         req_dur.record(&Context::new(), 0.42, &[
+//!         req_dur.record(0.42, &[
```

Comment:
doc example code was using outdated api that was not compiling. 
I'm guessing that `cargo test` has not been run here in a while.